src/main/java/racing/domain/game/RacingGameResult.java

```diff
@@ -16,19 +15,20 @@ public RacingGameResult(RacingCars racingCars, List<String> winners) {
     }
 
     public int getMaxPosition() {
-        return racingCars.getCars()
-                .stream()
-                .mapToInt(Car::getPosition)
-                .max()
-                .orElseThrow(() -> new IllegalArgumentException());
+        return RacingGame.findMaxPosition(racingCars);
     }
 
     public List<String> getWinnerCars(int maxCarPosition) {
-        return racingCars.getCars()
-                .stream()
-                .filter(list -> list.getPosition() == maxCarPosition)
-                .map(list -> list.getCarName())
-                .collect(Collectors.toList());
+        return RacingGame.findWinnersCars(racingCars, maxCarPosition);
+    }
+
+    public void resultRacing(int times) {
+        RacingGame racingGame = new RacingGame(racingCars);
+        List<RacingCars> racingCarsList = racingGame.race(times);
+
+        for (int i = 0; i < racingCarsList.size(); i++) {
+            ResultView.printResult(racingCars.getCars());
+        }
```

Comment:
```suggestion
        for (RacingCars cars : racingCarsList) {
            ResultView.printResult(cars.getCars());
        }
```
race의 결과로 받은 결과를 출력합니다 !